nva-datamodel-testutils/src/main/java/no/unit/nva/model/testing/RandomCurrencyUtil.java

```diff
@@ -0,0 +1,19 @@
+package no.unit.nva.model.testing;
+
+import java.util.Currency;
```

Comment:
Is this built-in in java?

Reply:
Yes, just to see familiar currency values in tests. Not considered how we want to validate user input in the API yet.